arrow/arrow.py

```diff
@@ -706,7 +725,11 @@ def float_timestamp(self):
 
         """
 
-        return self.timestamp + float(self.microsecond) / 1000000
+        # IDEA get rid of this in 1.0.0 and wrap datetime.timestamp()
+        # Or for compatibility retain this but make it call the timestamp method
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
This code is meant to catch the deprecation warning since it should not emit on this method correct?

Reply:
Yeah since the `float_timestamp` property calls `.timestamp` internally we need to suppress the warning but ensure the filter is restored afterwards.